Gemfile

```diff
@@ -8,7 +8,7 @@ group :development, :test do
   gem 'rake'
   gem 'pry-byebug'
   gem 'byebug'
-  gem 'rubocop'
+  gem 'rubocop', '~> 0.85.0'
```

Comment:
I don't think we should set a specific constraint here.

Reply:
If rubocop will fail in other versions, having a constraint makes sense, no?

Reply:
We only need to update `Gemfile.lock` for that.

(some projects do prefer to specify constraints for everything in the `Gemfile`, but since we're not doing that currently, I think we should following the existing convention).